_query-dsl/specialized/neural-sparse.md

```diff
@@ -10,12 +10,12 @@ nav_order: 55
 Introduced 2.11
 {: .label .label-purple }
 
-Use the `neural_sparse` query for vector field search in [neural sparse search]({{site.url}}{{site.baseurl}}/search-plugins/neural-sparse-search/). 
+Use the `neural_sparse` query for vector field search in [neural sparse search]({{site.url}}{{site.baseurl}}/search-plugins/neural-sparse-search/). Query can be performed using raw text or sparse vectors.
```

Comment:
```suggestion
Use the `neural_sparse` query for vector field search in [neural sparse search]({{site.url}}{{site.baseurl}}/search-plugins/neural-sparse-search/). The query can include both raw text and sparse vectors.
```

Reply:
The query should be used with either query text or tokens. Is it possible that the phrase `and` can cause confusion?

Reply:
@zhichao-aws: I adjusted my comment, let me know if it works for you.

Reply:
Works for me.